workflows/build.rs

```diff
@@ -0,0 +1,106 @@
+use anyhow::Result;
+use convert_case::{Case, Casing};
+use std::fs::File;
+use std::io::Write;
+use std::process::Command;
+use walkdir::WalkDir;
+use warp_workflows_types::Workflow;
+
+/// Generates Workflows as rust files from the yaml stored in /specs. Each Workflow is stored within
```

Comment:
This repo holds all of the _public_ workflows, so users won't need to copy this into the `.warp` directory.

This idea is that people who want to add onto the list of public workflows can add it in here. This repo is actually a cargo crate and will be imported as a library into our warp project, so the next time we release a new build, warp will automatically pick up the changes and include the new workflows.

It's essentially the exact same process that we're going to do for javascript: every time a workflow gets added, we'll create a new release of this library so that the consumer (either warp or commands.dev) picks up the changes.

Reply:
yeah that's entirely separate from this repo--this repo is the source of truth for workflows bundled into the app and into commands.dev. Workflows within the `.warp` directory are essentially private workflows that aren't globally exposed like these